cdragontoolbox/data.py

```diff
@@ -0,0 +1,62 @@
+from enum import Enum
+
+
+class Region(Enum):
```

Comment:
I don't see the purpose of an enum here.
Enum values and identifiers carry the same information, so it's just redundant. Having an enum rather than a plain string just make it more complicated to manipulate.

All we need is a list of known regions/langagues. Something like this:
```python
regions = ['br', 'cn', 'eun', ...]
```
It provides the basic features that we need: `some_string in regions`, `for region in regions`, `len(regions)`.

Reply:
Fixed